lib/mayaUsd/render/pxrUsdMayaGL/usdProxyShapeAdapter.cpp

```diff
@@ -171,10 +171,23 @@ bool PxrMayaHdUsdProxyShapeAdapter::_Sync(
         _renderTags.push_back(HdRenderTagTokens->guide);
     }
 
+    // Update the root transform used to render by the delagate.
+    // USD considare that the root prim transform is always the Identity matrix so that means
+    // the root transform define the root prim transform. When the real stage root is used to
+    // render this is not a issue because the root transform will be the maya transform.
+    // The problem is when using a primPath as the root prim, we are losesing
+    // the prim path world transform. So we need to set the root transform as the world
+    // transform of the prim used for rendering.
```

Comment:
Just a few typos in here:
considare -> considers
losesing -> losing

And looks like they're repeated in the render delegate below.